spec/lib/services/funding_eligibility_spec.rb

```diff
@@ -164,6 +173,32 @@
       end
     end
 
+    context "when there is no institution with at an approved ITT provider and they are a lead mentor" do
+      let(:institution) { nil }
+      let(:approved_itt_provider) { true }
+      let(:lead_mentor) { true }
+
+      context "and the course is NPQLTD" do
+        let(:course) { Course.all.find(&:npqltd?) }
+
+        it "is eligible" do
+          expect(subject.funded?).to be_truthy
+          expect(subject.funding_eligiblity_status_code).to eq :funded
+        end
+      end
+
+      context "and the course is not NPQLTD" do
+        Course.all.reject(&:npqltd?).each do |course|
+          let(:course) { course }
+
+          it "is not eligible for #{course.name}" do
+            expect(subject.funded?).to be_falsey
+            expect(subject.funding_eligiblity_status_code).to eq :not_lead_mentor_course
+          end
+        end
+      end
```

Comment:
I think we should have a spec here for if approved_itt_provider is false.

Reply:
Thats fair although in the feature flow it cant really get here because of the validations but from a spec level test it could happen.